docs/getting_started_jax.ipynb

```diff
@@ -0,0 +1,669 @@
+        "| `NamedSharding(mesh, PartitionSpec(None, 'model'))` | `{devices=[1,2,4]<=[4,2]T(1,0) last_tile_dim_replicate}`     |\n",
+        "| `UNSPECIFIED`    | nothing    |\n",
+        "\n",
+        "`UNSPECIFIED` is no sharding as expected since GSPMD will populate this during sharding propagation.\n",
+        "\n",
+        "Notice how all the axis names go away? While there is a 1:1 correspondance between `NamedSharding` and `HloShardingV2`, as a reader, it can be difficult to read. It is only more difficult once you introduce various axis names.\n",
+        "\n",
+        "Let's look at Shardy for comparison. To enable Shardy in JAX, simply enable the flag:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 6,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "_k7sNUIOQsxm",
+        "outputId": "92b45d74-cbaa-4aca-b300-eeec74b03ce0"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "module @jit_predict attributes {mhlo.num_partitions = 8 : i32, mhlo.num_replicas = 1 : i32} {\n",
+            "  sdy.mesh @mesh = <[\"data\"=4, \"model\"=2]>\n",
+            "  func.func public @main(%arg0: tensor<16x128xf32> {sdy.sharding = #sdy.sharding<@mesh, [{\"data\"}, {}]>}, %arg1: tensor<128x256xf32> {sdy.sharding = #sdy.sharding<@mesh, [{}, {\"model\"}]>}, %arg2: tensor<256x10xf32>) -> (tensor<16x10xf32> {jax.result_info = \"\"}) {\n",
+            "    %0 = stablehlo.tanh %arg0 : tensor<16x128xf32>\n",
+            "    %1 = stablehlo.dot_general %0, %arg1, contracting_dims = [1] x [0], precision = [DEFAULT, DEFAULT] : (tensor<16x128xf32>, tensor<128x256xf32>) -> tensor<16x256xf32>\n",
+            "    %2 = stablehlo.dot_general %1, %arg2, contracting_dims = [1] x [0], precision = [DEFAULT, DEFAULT] : (tensor<16x256xf32>, tensor<256x10xf32>) -> tensor<16x10xf32>\n",
+            "    %3 = stablehlo.sine %2 : tensor<16x10xf32>\n",
+            "    return %3 : tensor<16x10xf32>\n",
+            "  }\n",
+            "}\n",
+            "\n"
+          ]
+        }
+      ],
+      "source": [
+        "jax.config.update(\"jax_use_shardy_partitioner\", True)\n",
+        "run_in_out_shardings()\n",
+        "jax.config.update(\"jax_use_shardy_partitioner\", False)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "YAhl-fiERfOQ"
+      },
+      "source": [
+        "Now we have\n",
+        "\n",
+        "| JAX sharding    | Shardy sharding |\n",
+        "| -------- | ------- |\n",
+        "| `NamedSharding(mesh, PartitionSpec('data', None))`  | `#sdy.sharding<@mesh, [{\"data\"}, {}]>`    |\n",
+        "| `NamedSharding(mesh, PartitionSpec(None, 'model'))` | `#sdy.sharding<@mesh, [{}, {\"model\"}]>`     |\n",
+        "| `UNSPECIFIED`    | nothing    |\n",
```

Comment:
Ahh actually replace `UNSPEICIFIED` with `None` sorry!

Reply:
Replace `UNSPECIFIED` in the table with `None` as `UNSPECIFIED` isn't a public JAX API yet.